Plugins/BTCPayServer.RockstarDev.Plugins.TransactionCounter/CounterPluginSettings.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using BTCPayServer.RockstarDev.Plugins.TransactionCounter.ViewModels;
+
+namespace BTCPayServer.RockstarDev.Plugins.TransactionCounter;
+
+public class CounterPluginSettings
+{
+    public DateTime StartDate { get; set; }
+    public DateTime? EndDate { get; set; }
+    public bool Enabled { get; set; }
+    public bool AllStores { get; set; }
+    public string? Password { get; set; }
+    public string? CustomHtmlTemplate { get; set; }
+    public string? BackgroundVideoUrl { get; set; }
+    public string AdminUserId { get; set; }
+    public SelectedStoreViewModel[] SelectedStores { get; set; }
```

Comment:
Remove the `AllStores` and `SelectedStores` variables. Instead, keep only a `string` variable that holds a comma-separated list, which will be displayed using a `textarea`. For example:

`public string ExcludedStoreIds { get; set; }`

When selecting paid invoices, ensure that you select them from all stores that are not marked as Disabled or Archived, while also excluding stores whose IDs are listed in the `ExcludedStoreIds`.